inc/admin.php

```diff
@@ -80,7 +80,15 @@ function settings_screen() {
 	}
 ?>
 <div class="wrap">
-<?php if ( ! empty( $errors ) ) : ?>
+
+	<div class="notice notice-error adstxt-notice existing-adstxt" style="display: none;">
+		<p><strong><?php echo esc_html_e( 'Existing Ads.txt file found', 'ads-txt' ); ?></strong></p>
+		<p><?php echo esc_html_e( 'You will need to rename or remove the existing ads.txt file before you will be able to see any changes you make to ads.txt inside the WordPress admin.', 'ads-txt' ); ?></p>
```

Comment:
I would like for this to say something about **why** there's a difference between an actual file and what this plugin is doing, because I don't think that's very clear to most people. I'll think on exact language.

Reply:
How about something like:

> We detected a hard coded `ads.txt` file on your site. This file takes precedence over the plugin, preventing it from working correctly. Please remove this file to use `Ads-txt`.